tracardi/process_engine/action/v1/sort_dictionary.py

```diff
@@ -0,0 +1,121 @@
+from pydantic import validator
+
+from tracardi.service.plugin.domain.config import PluginConfig
+from tracardi.service.plugin.domain.register import Plugin, Spec, MetaData, Documentation, PortDoc, Form, FormGroup, \
+    FormField, FormComponent
+from tracardi.service.plugin.domain.result import Result
+from tracardi.service.plugin.runner import ActionRunner
+
+
+class Config(PluginConfig):
+    direction: str
+    data: str
+
+    @validator("direction")
+    def if_order_is_empty(cls, value):
+        if value == "":
+            raise ValueError("Direction cannot be empty")
+        return value
+
+    @validator("direction")
+    def if_order_has_valid_value(cls, value):
+        if value == "asc" or value == "desc":
+            return value
+        raise ValueError("Direction has invalid value")
```

Comment:
Could you add values that are possible (like: Possible values 'asc', 'desc') in the error message.